### PR TITLE
Support cesium rendering to new window

### PR DESCRIPTION
**Pull request by sgparrish**
Related Issue: #8235 

Change global `document` references to reference `ownerDocument` of relevant container element.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @sgparrish! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/sgparrish/cesium/blob/cesium-8235/CONTRIBUTORS.md) file!
* :x: Missing CLA.
    * Please send in a [Contributor License Agreement](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/cesium-8235/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by sgparrish** [comment]
My organization (Orbit Logic) should have a CLA already, I sent an email from the chain that originally set up the CLA to hopefully get added, but it's 18 months old, so I'm worried the point of contact is no longer correct.

**Comment by OmarShehata** [comment]
Thanks @sgparrish ! Can you forward that chain to `hello@cesium.com` ? I'll get it sorted on our side.

**Comment by sgparrish** [comment]
Thanks @OmarShehata . I forwarded the chain. Let me know if you still haven't received it.

**Comment by OmarShehata** [comment]
Thanks @sgparrish , I can confirm you're covered by the Orbit Logic corporate CLA.

File: Source/Widgets/FullscreenButton/FullscreenButtonViewModel.js [hunk 1]
```diff
@@ -23,13 +25,23 @@ define([
      * @alias FullscreenButtonViewModel
      * @constructor
      *
+     * @param {Element|String} container The DOM element or ID that will contain the widget.
      * @param {Element|String} [fullscreenElement=document.body] The element or id to be placed into fullscreen mode.
      */
-    function FullscreenButtonViewModel(fullscreenElement) {
+    function FullscreenButtonViewModel(container, fullscreenElement) {
```
Comment (mramato) [inline Source/Widgets/FullscreenButton/FullscreenButtonViewModel.js]: This is a breaking change, at the very least we need to first deprecate the parameter to support backwards compatibility during deprecation.  Another option is to make the `container` parameter second and have it default to document.body, that would avoid the breaking change entirely and everything would continue to work as-is.

File: Source/Widgets/Animation/Animation.js [hunk 2]
```diff
@@ -351,6 +351,8 @@ define([
         this._lastHeight = undefined;
         this._lastWidth = undefined;
 
+        var doc = container.ownerDocument;
```
Comment (mramato) [inline Source/Widgets/Animation/Animation.js]: Sorry to be picky, but can we call this variable `ownerDocument` just so its crystal clear to the reader that `doc !== document`?

Comment (mramato) [reply Source/Widgets/Animation/Animation.js]: Same comment throughout, just want to avoid future confusion and silly mistakes.

**Comment by mramato** [comment]
Sorry for the delayed review here, and thanks for the PR @sgparrish.  I think those are the only comments I have.  Please merge in master and bump when this is ready.  Thanks again!

**Comment by sgparrish** [comment]
All comments should be addressed, and merge conflicts resolved.

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @sgparrish! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @sgparrish! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by sgparrish** [comment]
Not sure if anyone's had a chance to look this over again, but it would be nice to get this into an actual release in the next couple months.

**Comment by hpinkos** [comment]
@mramato can we merge this?

**Comment by mramato** [comment]
I'm sorry for the slow roll here @sgparrish 

I merged in master and made a minor JSDoc fix.  Once CI passes this should be good to merge and will ship on Monday as part of 1.67.

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @sgparrish! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by mramato** [comment]
I'm not sure what kind of brain fart I had in this never getting merged.  I just merged in master again and will get this merged today for 1.68 on Wednesday.  Sorry for totally dropping the ball here.

**Pull request merged**